apps/web/lib/web/templates/opportunities/opportunities.html.eex

```diff
@@ -1,42 +1,48 @@
 <svg style="display: none;">
   <symbol viewBox="0 0 100 125" id="icon-tag"><title>tag</title>
-    <path d="M96.4,3.6c-0.5-0.5-1.3-0.9-2.1-0.9c0,0,0,0,0,0l-38,0.9C53,3.8,50,5.1,47.5,7.4L7.3,47.6c-2.9,2.9-4.5,6.9-4.5,10.9   c0,3.7,1.5,7.3,4.2,10.2l24.6,24.6c2.8,2.8,6.4,4.1,10.1,4.1c3.9,0,7.9-1.5,10.9-4.5l40.1-40.2c2.4-2.4,3.8-5.6,3.8-8.9l0.9-38.1   C97.3,4.9,97,4.2,96.4,3.6z M75.4,36.8c-1.6,1.6-3.9,2.6-6.1,2.6c-2.1,0-4.4-1-5.9-2.6c-1.6-1.4-2.5-3.6-2.5-6c0-2.2,0.9-4.4,2.5-6   c3.4-3.3,8.8-3.3,12,0c1.6,1.4,2.5,3.6,2.5,6C77.9,32.9,77,35.2,75.4,36.8z"/>
+  <path d="M96.4,3.6c-0.5-0.5-1.3-0.9-2.1-0.9c0,0,0,0,0,0l-38,0.9C53,3.8,50,5.1,47.5,7.4L7.3,47.6c-2.9,2.9-4.5,6.9-4.5,10.9   c0,3.7,1.5,7.3,4.2,10.2l24.6,24.6c2.8,2.8,6.4,4.1,10.1,4.1c3.9,0,7.9-1.5,10.9-4.5l40.1-40.2c2.4-2.4,3.8-5.6,3.8-8.9l0.9-38.1   C97.3,4.9,97,4.2,96.4,3.6z M75.4,36.8c-1.6,1.6-3.9,2.6-6.1,2.6c-2.1,0-4.4-1-5.9-2.6c-1.6-1.4-2.5-3.6-2.5-6c0-2.2,0.9-4.4,2.5-6   c3.4-3.3,8.8-3.3,12,0c1.6,1.4,2.5,3.6,2.5,6C77.9,32.9,77,35.2,75.4,36.8z"/>
```

Comment:
Maybe leave a comment explaining what this is? Cuz its greek to me and seemingly important.

Reply:
This is the tag SVG @burden found.  Maybe we can store it in the SCSS files?  @burden do you know?

Reply:
That makes sense, I just suck at FE dev so bad I didnt realize it at a glance. I've seen this before, I don't think its generally considered poor form from what I have seen.

Reply:
Do we still need this SVG? that actually displays: none;

Reply:
This SVG declaration is the tag icon. It's referenced in app.scss https://github.com/elixirschool/extracurricular/blob/cd48da636f52d7cfa52a6dcbd0e57b9ada48c1c8/apps/web/assets/css/app.scss#L59 where it changes the default display from none to "inline".

Reply:
@doomspork SVG cannot go into css, because it's markup.

Reply:
Sweet, thanks @burden 👍